ONBOARDING.md

```diff
@@ -1,50 +1,69 @@
 # Node.js build WG onboarding
 
-This document outlines what things future members of the build WG should be
-acquainted with. It might be expanded into additional documentation down
-the road.
+This document is an outline of the things we tell new Build WG members at their
+onboarding session.
 
-### General information
+### Before the onboarding session
 
-  - Responsibilities
-  - Members/"who does what"/interest areas
-  - Technologies and software in use
+* Email the new member and obtain their public GPG key
+* Add them to the `test` group in `nodejs-private/secrets`
+* Add them to the `nodejs/build` and `nodejs-private/build` GitHub teams
+* Ask them to start looking over `services.md` and `jenkins-guide.md`,
+  and to try and have any questions ready
+* Ask the onboardee to install:
+  * Ruby and the `dotgpg` gem
+  * Python 2.7 and Ansible
 
+### General information
 
-### CI/Jenkins
+* Responsibilities: The Build WG manages everything in
+  [`doc/services.md`](doc/services.md)
+* Members/"who does what"/interest areas
+* Technologies and software in use
 
-  - `nodejs/build` ci access
-  - CI and CI-Release
-  - Setting up a node
-  - `iptables`/infra group
-  - Jobs
+### CI/Jenkins
 
+* Public CI (ci.nodejs.org)
+  * `nodejs/jenkins-admins` are administrators
+  * Tests Node.js itself, and several other Foundation projects
+* Release CI (ci-release.nodejs.org)
+  * `nodejs/jenkins-release-admins` are administrators
+  * What creates and publishes all releases of Node.js
+* General trend is try and use Jenkins pipelines as much as possible,
+  store all bash scripts within `jenkins/scripts`
 
 ### Secrets
 
-  - Different access groups
-  - Using dotGPG
-
+* All production secrets are stored in the `build` directory of the
+[`nodejs-private/secrets` repository][]
+* The access groups are explained in [`doc/access.md`](doc/access.md)
+* Say you want to decrypt the file `a.txt`, you'd run `dotgpg cat a.txt`
+* Follow the instructions in [`doc/ssh.md`](doc/ssh.md) to setup the secrets
+  repo and SSH accesslocally
 
 ### Ansible
 
-  - Build repo layout
-  - Adding a node
-  - Running the playbook
-  - Common tasks
-
-
-### Build admin
-
-  - Common tasks
-    - Restarting a node/jenkins
-    - Fixing issues (trailing node processes, etc)
-  - Windows access
-
```

Comment:
This section still seems relevant to me. Of course, how much is talked about each point depends on the experience of the person doing the onboarding and the interests of the person being onboarded.

Reply:
The restarting node/fixing issues stuff is part of `doc/jenkins-guide.md`, but will add back Windows, didn't mean to delete that